public/script.js

```diff
@@ -0,0 +1,56 @@
+const sourceTextContainer = document.querySelector('.source-text-container')
+
+const spanArray = []
+
+const h1 = document.querySelector('h1')
+
+function populateSourceText(data) {
+  sourceTextContainer.innerHTML = ''
+  spanArray.splice(0, spanArray.length)
+  
+  data.forEach((text) => {
+    const p = document.createElement('p')
+    text.split(" ").forEach((word) => {
+      const span = document.createElement('span')
+      span.innerText = word
+      span.tabIndex = "4"
+      spanArray.push(span)
+      p.appendChild(span)
+      p.appendChild(document.createTextNode(" "))
+    })
+    sourceTextContainer.appendChild(p)
+  })
+
+  // puts the first word in the title for screen reader
+
+  let firstWord = document.querySelector('#first-word')
+  if (firstWord !== null) {
+    firstWord.remove()
+  }
+  firstWord = document.createElement('span')
+  firstWord.id = 'first-word'
+  firstWord.classList.add('sr-only')
+  firstWord.innerText = 'The first word is: ' + spanArray[0].innerText
+  h1.appendChild(firstWord)
+  h1.blur()
+  h1.focus()
```

Comment:
why do you blur it and then immediately focus it?

Reply:
this is realllllly weird. When we went to focus first, the screen reader didn't read out an extra span that we added via javascript signposting the first word of the source text, but when we blurred and then focused it did